core/system/inc/mpu/vmpu_freescale_k64.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __VMPU_FREESCALE_K64_H__
+#define __VMPU_FREESCALE_K64_H__
+
+/* MPU fault status codes */
+#define VMPU_FAULT_NONE     (-1)
+#define VMPU_FAULT_MULTIPLE (-2)
+
+/** Get the slave port number from the MPU->CESR SPERR field.
+ * @returns The slave port number, or a negative value to signal an error
+ * @retval \ref VMPU_FAULT_NONE     No MPU violation found
+ * @retval \ref VMPU_FAULT_MULTIPLE Multiple MPU violations found
+ */
+UVISOR_FORCEINLINE int vmpu_fault_get_slave_port(void)
```

Comment:
Yup, that was the reason. -1 means no MPU violation, -2 means multiple violations.